modules/fundamental/tests/issues/oom.rs

```diff
@@ -2,22 +2,22 @@
 
 use std::str::FromStr;
 use test_context::test_context;
-use test_log::test;
-use tracing::instrument;
 use trustify_common::id::Id;
 use trustify_module_fundamental::sbom::service::SbomService;
-use trustify_test_context::TrustifyContext;
+use trustify_test_context::{TrustifyContext, flame::setup_global_subscriber};
 
 #[test_context(TrustifyContext, skip_teardown)]
-#[test(tokio::test)]
-#[instrument]
+#[tokio::test]
```

Comment:
just wondering out load if we could make test dependent on dataset (maybe with feature enabling via cfg annotation)

Reply:
Yea, that would be nice.